### ISSUE TITLE
📝 Spec issue: failing WPT test cases

### ISSUE THREAD

**Issue by drwpow** [drwpow/html-aria#58]
### Specification reference(s)

From [this comment](https://github.com/drwpow/html-aria/issues/25#issuecomment-2727546397):

> Aside from the above, there are now also 16 additional WPT cases failing when using html-aria (suspect aria-query and other alternatives may well fail as well). See the test cases listed against roles-contextual.html in https://github.com/guidepup/virtual-screen-reader/blob/main/test/wpt-jsdom/to-run.yaml#L367:
>
>    el-aside-in-section-title
    el-img-no-name
    el-img-empty-alt-title
    el-section-title
    el-aside-in-section-aria-label-whitespace
    el-aside-in-section-aria-labelledby-non-existing
    el-aside-in-section-aria-labelledby-empty
    el-aside-in-section-aria-labelledby-whitespace
    el-img-empty-alt-aria-label-whitespace
    el-img-empty-alt-aria-labelledby-non-existing
    el-img-empty-alt-aria-labelledby-empty
    el-img-empty-alt-aria-labelledby-whitespace
    el-section-aria-label-whitespace
    el-section-aria-labelledby-non-existing
    el-section-aria-labelledby-empty
    el-section-aria-labelledby-whitespace
>
> You can find these HTML test cases in https://github.com/web-platform-tests/wpt/blob/master/html-aam/roles-contextual.html (introduced a week ago in https://github.com/web-platform-tests/wpt/pull/50679 as part of the 2025 browser interop effort)

Align with the failing WPT tests and/or provide sufficient reasoning why the output can’t align (as of now there is unintentional drift)

### Output

Failed WPT tests

### Contributing

- [x] I am willing to open a Pull Request
